src/components/Table/Internal/octable.module.scss

```diff
@@ -259,6 +269,14 @@
       text-align: left;
     }
 
+    .table-cell-fix-left-first,
+    .table-cell-fix-left-last,
+    .table-cell-fix-right-first,
+    .table-cell-fix-right-last {
+      overflow: clip;
+      overflow-clip-margin: $space-xs;
```

Comment:
i've never heard of this property, and thats probably because it has some of the worst browser support I've ever seen 😆 
https://caniuse.com/?search=overflow-clip-margin

Whats the downside here for browsers lacking support? peek-a-boo of corners while scrolling?

Reply:
The box shadow would be hidden